src/components/collapsible/Collapsible.js

```diff
@@ -107,7 +113,7 @@ class Collapsible extends React.PureComponent<Props, State> {
 
         return (
             <div className={sectionClassName}>
-                <div className={buttonClassName}>
+                <div className={buttonClassName} onBlur={onBlur} onFocus={onFocus}>
```

Comment:
BTW, whats the use case of onBlur and onFocus? Just need them for something in parent component? Also should they be on the button instead, since thats the focus target.

Reply:
after the CSS adjustments, I don't see a use case for these functions at least for now. have removed it